src/sentry/static/sentry/app/components/commitLink.jsx

```diff
@@ -13,10 +13,16 @@ class CommitLink extends React.Component {
 
   getCommitUrl = () => {
     // TODO(jess): move this to plugins
-    if (this.props.repository.provider.id === 'github') {
+    if (
+      ['github', 'integrations:github'].indexOf(this.props.repository.provider.id) != -1
```

Comment:
You can use [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)